lib/constants/error-messages.ts

```diff
@@ -33,6 +33,7 @@ export const REPOSITORY_NOT_FOUND = 'not-found';
 export const REPOSITORY_NO_PACKAGE_FILES = 'no-package-files';
 export const REPOSITORY_RENAMED = 'renamed';
 export const REPOSITORY_UNINITIATED = 'uninitiated';
+export const REPOSITORY_ONBOARDING_SKIPPED = 'repository-onboarding-skipped'; // skip processing of onboarding repos that are up to date
```

Comment:
Throwing an _error_ for a non-error case seems wrong

Reply:
it's much easier than passing that information all the way up

Reply:
I really don't think a repository being "up to date" should trigger a thrown error, even if it's easy. I would like our code to have less thrown errors